src/preprocess.js

```diff
@@ -95,21 +101,27 @@ async function replaceGeometryIndexes() {
 }
 
 function updateEncodingInner(filename) {
-  return new Promise((resolve, reject) => {
+  return new Promise((resolve) => {
     const filePath = path.join(__dirname, "..", "data", filename);
-    const tempPath = `${filePath}.tmp`;
+    const destPath = path.join(__dirname, "..", "processed", filename);
```

Comment:
Output utf8 data into `processed` to make this function work reliably and intuitively.